packages/ui-components/src/image.ts

```diff
@@ -63,3 +63,15 @@ class ConfinedImage extends Component<ConfinedImageProps, ConfinedImageState> {
 }
 
 export { ConfinedImage };
+
+export function MacrostratImage({ src, className, width, height }) {
+  const srcWithAddedPrefix =
+    "https://storage.macrostrat.org/assets/web/" + src;
+  return h("img", { src: srcWithAddedPrefix, className, width, height });
+}
+
+export function RockdImage({ src, className, width, height }) {
+  const srcWithAddedPrefix =
+    "https://storage.macrostrat.org/assets/rockd/" + src;
+  return h("img", { src: srcWithAddedPrefix, className, width, height });
+}
```

Comment:
These are essentially wrappers for configuration that we could reset at any time, and should be stored in their respective repositories, not the shared components library. Maybe a more general `AppImage` component that got its src values from a react context would be useful? But these are simple to keep in the [`UW-Macrostrat/web`](https://github.com/UW-Macrostrat/web) repo as well.

It's fine to have this type of code oriented to a specific app in the stories, but we should ideally try to avoid it in the web components proper.